test/unit/membership_test.js

```diff
@@ -72,6 +72,33 @@ testRingpop('change with higher incarnation number results in leave override', f
     assert.equals(member.status, Member.Status.leave, 'results in leave');
 });
 
+testRingpop('change that overrides the local status should be overwritten to a change that reincarnates the node', function t(deps, assert) {
+    var ringpop = deps.ringpop;
+    var membership = deps.membership;
+
+    var member = membership.findMemberByAddress(ringpop.whoami());
+    assert.equals(member.status, Member.Status.alive, 'member starts alive');
+
+    var applied = membership.update([{
+        source: "192.0.2.1:1234",
```

Comment:
Can we assert somewhere in the test that `"192.0.2.1:1234" != ringpop.whoami()` to make sure line 98 is testing what we expect?